.github/workflows/ci.yml

```diff
@@ -6,103 +6,48 @@ on:
     strategy:
       fail-fast: false
       matrix:
         suite:
-          - { path: bundler, name: bundler1 }
-          - { path: bundler, name: bundler2 }
-          - { path: cargo, name: cargo }
-          - { path: common, name: common }
-          - { path: composer, name: composer }
-          - { path: docker, name: docker }
-          - { path: elm, name: elm }
-          - { path: git_submodules, name: git_submodules }
-          - { path: github_actions, name: github_actions }
-          - { path: go_modules, name: go_modules }
-          - { path: gradle, name: gradle }
-          - { path: hex, name: hex }
-          - { path: maven, name: maven }
-          - { path: npm_and_yarn, name: npm_and_yarn }
-          - { path: nuget, name: nuget }
-          - { path: omnibus, name: omnibus }
-          - { path: python, name: python }
-          - { path: python, name: python_slow }
-          - { path: pub, name: pub }
-          - { path: terraform, name: terraform }
-    env:
-      BASE_IMAGE: ubuntu:18.04
-      CORE_IMAGE: dependabot/dependabot-core
-      CORE_BRANCH_IMAGE: ghcr.io/dependabot/dependabot-core-branch
-      CORE_CI_IMAGE: ghcr.io/dependabot/dependabot-core-ci
-      CODE_DIR: /home/dependabot/dependabot-core
+          - { path: bundler, name: "Bundler" }
+          - { path: cargo, name: "Cargo" }
+          - { path: common, name: "Common" }
+          - { path: composer, name: "Composer" }
+          - { path: docker, name: "Docker" }
+          - { path: elm, name: "Elm" }
+          - { path: git_submodules, name: "Git Submodules" }
+          - { path: github_actions, name: "GitHub Actions" }
+          - { path: go_modules, name: "Go Modules" }
+          - { path: gradle, name: "Gradle" }
+          - { path: hex, name: "Hex" }
+          - { path: maven, name: "Maven" }
+          - { path: npm_and_yarn, name: "npm and Yarn" }
+          - { path: nuget, name: "NuGet" }
+          - { path: python, name: "Python" }
+          - { path: pub, name: "Pub" }
+          - { path: terraform, name: "Terraform" }
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Prepare BRANCH_REF environment variable
-        run: echo "BRANCH_REF=$(echo '${{ github.ref }}' | sed -E 's/[^A-Za-z0-9]+/-/g')" >> $GITHUB_ENV
-      - name: Log in to GHCR
-        run: |
-          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
-      - name: Build dependabot-core image for branch
-        env:
-          DOCKER_BUILDKIT: 1
-        run: |
-          docker build \
-            -t "$CORE_IMAGE:latest" \
-            -t "$CORE_BRANCH_IMAGE:$BRANCH_REF" \
-            --build-arg BUILDKIT_INLINE_CACHE=1 \
-            --cache-from "$BASE_IMAGE" \
-            --cache-from "$CORE_IMAGE:latest" \
-            --cache-from "$CORE_BRANCH_IMAGE:$BRANCH_REF" \
-            .
-      - name: Push dependabot-core-branch image to GHCR
```

Comment:
We do sometimes use these branch builds for testing. I don't think it's worth keeping in here since it's a lot simpler without it! We could use a tag or branch prefix to trigger branch build in the docker.yml if we need it.